qgan_gradient_fix-6d2dd8f1b471d8c0.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
```

Comment:
This reno note needs to be moved to replace the generic template one of the same note in the releasenotes/notes folder

Reply:
Sure, will do.